packages/react-icons/src/utils/constants.tsx

```diff
@@ -0,0 +1,2 @@
+export const FILLED_CLASSNAME = "fui-Icon-filled";
+export const REGULAR_CLASSNAME = "fui-Icon-regular";
```

Comment:
I forgot to mention this, but let also use the same naming:

```tsx
export const iconFilledClassName = "fui-Icon-filled";
export const iconRegularClassName = "fui-Icon-regular";
```

https://github.com/microsoft/fluentui/blob/42c69701f6af74ab511549343c01a88d1889cfd3/packages/react-card/src/components/Card/useCardStyles.ts#L5